prisma/schema.prisma

```diff
@@ -8,14 +8,14 @@ datasource db {
 }
 
 model User {
-  id             Int       @id @default(autoincrement())
-  name           String?   @db.VarChar(255)
-  role           role      @default(USER)
-  email          String?   @unique @db.VarChar(255)
-  email_verified DateTime? @db.Timestamptz(6)
-  image          String?
-  created_at     DateTime  @default(now()) @db.Timestamptz(6)
-  updated_at     DateTime  @default(now()) @db.Timestamptz(6)
+  id            Int       @id @default(autoincrement())
+  name          String?   @db.VarChar(255)
+  role          role      @default(USER)
+  email         String?   @unique @db.VarChar(255)
+  emailVerified DateTime? @map("email_verified") @db.Timestamptz(6)
```

Comment:
Is there a reason why this is camel case but `created_at` and `update_at` are snake case?

Reply:
Good catch, lol. 

It's a weird `next-auth` condition that expects it to be camelCased. If you change the base property to snake_case it just wont work.